.github/workflows/tests.yml

```diff
@@ -0,0 +1,44 @@
+name: Reportengine tests
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.12"]
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@v3
+      with:
+        activate-environment: test
+        use-mamba: true
+
+    - name: Update Environment
+      shell: bash -l {0}
+      run: |
+        conda env update -n test -f environment.yml
```

Comment:
Oh I thought I wrote a comment on this as well. Anyway, why do you not just do `conda install pandoc` followed by `pip install .` instead of declaring the dependencies in an `evironment.yml`?

Reply:
Hi Roy, sorry for the very late reply. I somehow forgot about this PR.

I think that having the environment.yml has the advantage of allowing the user to install reportengine and creating a conda env at the same time:

`conda env create -f environment.yml`

If we agree on keeping the env file, then why not using it directly in the CI?

Reply:
Also a long time ago for me... 

Anyway, the dependencies are already defined in meta.yaml for installing with conda and pyproject.toml for python. In the test you already build the conda package, so you could create an env using the local package and run the test on that. That way you don't have to define the dependencies in yet another place. 

Also `conda install --only-deps` exists, so people could use that to install deps instead of environment.yaml.

Reply:
Can you be a bit more specific please?
I am not sure the conda package is built in this test. I am only setting up miniconda.